dangerfile.js

```diff
@@ -86,12 +86,12 @@ const componentSourceFilesModified = fileChanges.some(
 );
 
 const snapshotsModified = fileChanges.some(
-  (filePath) => filePath.endsWith('.js.snap') || filePath.endsWith('.ts.snap'),
+  (filePath) => filePath.endsWith('.js.snap') || filePath.endsWith('.tsx.snap'),
 );
 
 if (componentSourceFilesModified && !snapshotsModified) {
   warn(
-    "Package source files (e.g. `packages/package-name/src/Component.js`) were updated, but snapshots weren't. Have you checked that the tests still pass?",
+    "Package source files (e.g. `packages/package-name/src/Component.tsx`) were updated, but snapshots weren't. Have you checked that the tests still pass?",
```

Comment:
My thinking was that as we know we're aiming for TS only this helps to keep that mentality in mind from step one (this PR) onwards? We're now in 'Flow and plain JS is legacy', matching the internal standard.

WDYT?

>  I will also check with the team for alignment and agreement of the decision doc that has been added!

Thanks!

Reply:
I think for this whilst this is the goal - if we are making some fixes or modifications before doing a full component TS migration it would still be a js file so would like to keep it from being strict to avoid people feeling like they need to make the whole component TS compatible for a small fix

Reply:
https://github.com/Skyscanner/backpack/pull/2535/commits/d74777dedce3b7af687f9022afc072bf7470ecb7